_observing-your-data/alerting/dashboards-alerting.md

```diff
@@ -9,84 +9,308 @@ nav_order: 50
 Introduced 2.9
 {: .label .label-purple }
 
-Create, manage, and take action on your alerts in a single, consolidated view and identify and resolve issues quickly. Use the **Dashboard** interface to:
-
-- Set up, add, and adjust rules and conditions that trigger alerts and notifications.
-- Create graphs that show trends and patterns and build intuitive dashboards to stay informed of important metrics and data points in real time.
-- Monitor your alerts in one place with at-a-glance views.
-
-The following image gives you a snapshot of the Dashboard interface. 
-
-<img src="{{site.url}}{{site.baseurl}}/images/dashboards/alerting-dashboard.png" alt="Example alerting visualization" width="800" height="800">
+OpenSearch Dashboards offers a unified interface to efficiently create, manage, and respond to alerts within the **Dashboards** application. It enables rapid issue identification and resolution by visualizing alerts and related data on a centralized dashboard.
 
 ## Getting started 
 
-Before getting started, you must have:
+Before working with alerting dashboards and visualizations, ensure that you have:
 
 - Installed OpenSearch and OpenSearch Dashboards version 2.9 or later. See [Installing OpenSearch]({{site.url}}{{site.baseurl}}/install-and-configure/install-opensearch/index/).
-- Installed the Alerting and Notifications Dashboards plugins. See [Managing OpenSearch Dashboards plugins]({{site.url}}{{site.baseurl}}/install-and-configure/install-dashboards/plugins/) to get started.
+- Installed the Alerting, Anomaly Detection, and Notifications Dashboards plugins. See [Managing OpenSearch Dashboards plugins]({{site.url}}{{site.baseurl}}/install-and-configure/install-dashboards/plugins/).
+- Started your local environment. The [OpenSearch Playground](https://playground.opensearch.org/app/home) is read-only, so you should use your local environment to perform the steps in the following tutorials.
 
 ## Configuring admin settings
 
-Users can only access, create, or manage alerts for resources for which they have permissions. Access to alerting dashboards and visualizations is controlled by OpenSearch and OpenSearch Dashboards permissions. It is enabled by default and appears as a feature under **Dashboards Management** > **Advanced Settings** > **Visualization**. If the setting is disabled, it does not appear. You can disable the setting at the cluster level in the `opensearch-dashboards.yml` file.
+Access to alerting dashboards and visualizations is controlled by OpenSearch and OpenSearch Dashboards permissions. You can only access, create, or manage alerts on resources for which you have appropriate permissions.
```

Comment:
Should we specify more in-depth what permissions are needed exactly? Or remove this.

Reply:
@AWSHurneyt Please provide feedback on the above question.

Reply:
More information about alerting security can be found here. Would hyperlinking to this page be appropriate?
https://opensearch.org/docs/latest/observing-your-data/alerting/security/